automated/linux/usb/test-usb-2.sh

```diff
@@ -0,0 +1,41 @@
+#!/bin/bash
+
+PATH_DEV="/dev/sda1"
+PATH_MOUNT="/mnt/lava_usb_test"
+
+lsblk | grep sda1
+if [ $? -eq 0 ]
+then
+    #Flash disk will be formatted
+    mkfs.ntfs "$PATH_DEV"
+
+    if [ $? -eq 0 ]
+    then
+        umount "$PATH_MOUNT"
+        mkdir "$PATH_MOUNT"
```

Comment:
will fail if the folder already exists -> will the test then fail? if the test will fail, check the folder for existence and create it only, when the folder isn't there or add || true to the line